packages/flutter/lib/src/gestures/force_press.dart

```diff
@@ -0,0 +1,260 @@
+
+import 'dart:ui' show Offset;
+
+import 'arena.dart';
+import 'constants.dart';
+import 'events.dart';
+import 'recognizer.dart';
+
+enum _ForceState {
+  ready,
+  possible,
+  accepted,
+  started,
+  peaked,
+}
+
+/// Details object for callbacks that use [GestureForcePressStartCallback],
```

Comment:
Yes? But the pressure value will be sent in onUpdate. In this way, onStart and the first onUpdate will sent in the same frames. It's the way it's done for monodrag.dart so I thought it would make sense to replicate the behavior on this recognizer.

Reply:
I decided to just add the pressure into all the calls – so you don't have to have an onUpdate to get the pressure if you really only want the onStart call.